src/main/resources/application.yml

```diff
@@ -38,6 +38,7 @@ server:
     remoteip:
       remote-ip-header: x-forwarded-for
       protocol-header: x-forwarded-proto
+      internal-proxies: 10\.\d{1,3}\.\d{1,3}\.\d{1,3}|192\.168\.\d{1,3}\.\d{1,3}|169\.254\.\d{1,3}\.\d{1,3}|127\.\d{1,3}\.\d{1,3}\.\d{1,3}|172\.1[6-9]{1}\.\d{1,3}\.\d{1,3}|172\.2[0-9]{1}\.\d{1,3}\.\d{1,3}|172\.3[0-1]{1}\.\d{1,3}\.\d{1,3}|0:0:0:0:0:0:0:1|::1|100\.6[4-9]\.\d{1,3}\.\d{1,3}|100\.[7-9][0-9]{1}\.\d{1,3}\.\d{1,3}|100\.1[0-1][0-9]{1}\.\d{1,3}\.\d{1,3}|100\.12[0-7]\.\d{1,3}\.\d{1,3}
```

Comment:
This is expanded into
```
10\.\d{1,3}\.\d{1,3}\.\d{1,3}
192\.168\.\d{1,3}\.\d{1,3}
169\.254\.\d{1,3}\.\d{1,3}
127\.\d{1,3}\.\d{1,3}\.\d{1,3}
172\.1[6-9]{1}\.\d{1,3}\.\d{1,3}
172\.2[0-9]{1}\.\d{1,3}\.\d{1,3}
172\.3[0-1]{1}\.\d{1,3}\.\d{1,3}
0:0:0:0:0:0:0:1
::1
100\.6[4-9]\.\d{1,3}\.\d{1,3}
100\.[7-9][0-9]{1}\.\d{1,3}\.\d{1,3}
100\.1[0-1][0-9]{1}\.\d{1,3}\.\d{1,3}
100\.12[0-7]\.\d{1,3}\.\d{1,3}
```
so the top lot are the default in the RemoteIpValve and the bottom lot are 100.64/10, which translates to 100.64.0.0 -> 10.127.255.255